mmyolo/models/necks/base_yolo_neck.py

```diff
@@ -33,7 +33,7 @@ class BaseYOLONeck(BaseModule, metaclass=ABCMeta):
 
     def __init__(self,
                  in_channels: List[int],
-                 out_channels: int,
+                 out_channels: List[int],
```

Comment:
out_channels: Union[List[int], int]

Reply:
Is it necessary for us to support `int`, which means that the output channels are all the same?